gobblin-cluster/src/main/java/org/apache/gobblin/cluster/GobblinHelixJobLauncher.java

```diff
@@ -539,7 +537,7 @@ private void deleteWorkUnitFromStateStore(String workUnitId, ParallelRunner stat
     Path workUnitFile = new Path(workUnitFilePath);
     final String fileName = workUnitFile.getName();
     final String storeName = workUnitFile.getParent().getName();
-    if (fileName.endsWith(MULTI_WORK_UNIT_FILE_EXTENSION)) {
+    if (fileName.endsWith(JobLauncherUtils.MULTI_WORK_UNIT_FILE_EXTENSION)) {
```

Comment:
I see this sort of check a lot in this PR, does it make sense to make it a reusable function of JobLauncherUtils where we check if a given path/string is a multiworkunit file?

Reply:
good call!  I added